commitment/commitment_test.go

```diff
@@ -438,8 +487,16 @@ func TestSplitCommitment(t *testing.T) {
 	outPoint := wire.OutPoint{}
 	genesisNormal := asset.RandGenesis(t, asset.Normal)
 	genesisCollectible := asset.RandGenesis(t, asset.Collectible)
-	groupKeyNormal := asset.RandGroupKey(t, genesisNormal)
-	groupKeyCollectible := asset.RandGroupKey(t, genesisCollectible)
+	normalProtoAsset := randAsset(t, genesisNormal, nil)
+	collectibleProtoAsset := randAsset(t, genesisCollectible, nil)
+	groupKeyNormal := asset.RandGroupKey(t, genesisNormal, normalProtoAsset)
+	groupKeyCollectible := asset.RandGroupKey(
+		t, genesisCollectible, collectibleProtoAsset,
```

Comment:
Why's this need both the genesis and the actual asset? The asset includes the genesis.

Reply:
Ah nvm, with the way that `DeriveGroupKey` func works, it always wants to sign, and during reissuance, you're signing the new asset.